package.json

```diff
@@ -38,6 +41,7 @@
     "moment": "2.12.0",
     "read": "1.0.7",
     "request": "2.69.0",
+    "semver": "5.1.0",
```

Comment:
We're already pulling `semver` as part of some other dependency.

Runtime warning helps notify server owners, I guarantee you some of them won't be reading changelogs.